frontend/src/layout/navigation-3000/sidepanel/panels/activation/SidePanelActivation.tsx

```diff
@@ -280,16 +287,27 @@ const ActivationTask = ({
                     )}
                     <p className="m-0 font-semibold">{title}</p>
                 </div>
-                {isActive && canSkip && (
-                    <LemonButton
-                        size="xsmall"
-                        type="secondary"
-                        className="h-6 font-semibold text-muted-alt activation-task-skip"
-                        onClick={handleSkip}
-                    >
-                        Skip
-                    </LemonButton>
-                )}
+                {isActive &&
+                    canSkip &&
+                    (skipped ? (
+                        <LemonButton
+                            size="xsmall"
+                            type="secondary"
+                            className="h-6 font-semibold text-muted-alt activation-task-skip"
+                            onClick={handleUnskip}
+                        >
+                            Unskip?
```

Comment:
I think this would look cleaner without the `?`


```suggestion
                            Unskip
```

Reply:
I just think it looks a little confusing with the `?`. But, up to you. I'll approve it either way, it's not a deal breaker to me.